common/dynamicconfig/config_test.go

```diff
@@ -41,7 +41,7 @@ func TestConfigSuite(t *testing.T) {
 	suite.Run(t, s)
 }
 
-func (s *configSuite) SetupSuite() {
+func (s *configSuite) SetupTest() {
```

Comment:
key changes were leaking between tests.  doesn't seem like that's even remotely desired in these, so now each test gets a new value.